.github/workflows/docker-tag-push.yml

```diff
@@ -48,7 +48,6 @@ jobs:
           image: ${{ inputs.image }}
           variant: ${{ inputs.variant }}
           platform: aarch64
-        if: ${{ !contains(inputs.variant, 'cuda') }}
```

Comment:
You either have to make it work for pytorch-cuda11, or do not download it here, otherwise it will fail